benchpress/install_minebench.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+set -x
+
```

Comment:
Can we add a section for prerequisites and add the libraries we assume are going to be installed.

Maybe use our test host as baseline and then add any additional libraries that were installed before we can run this benchmark